Jellyfin.Server.Implementations/Users/UserManager.cs

```diff
@@ -198,18 +214,10 @@ public User CreateUser(string name)
 
             using var dbContext = _dbProvider.CreateContext();
 
-            // TODO: Remove after user item data is migrated.
-            var max = dbContext.Users.Any() ? dbContext.Users.Select(u => u.InternalId).Max() : 0;
+            var newUser = await CreateUserInternalAsync(name, dbContext).ConfigureAwait(false);
 
-            var newUser = new User(
-                name,
-                _defaultAuthenticationProvider.GetType().FullName,
-                _defaultPasswordResetProvider.GetType().FullName)
-            {
-                InternalId = max + 1
-            };
-            dbContext.Users.Add(newUser);
-            dbContext.SaveChanges();
+            await dbContext.Users.AddAsync(newUser).ConfigureAwait(false);
```

Comment:
Adding to the context should be done synchronously, since this is not an IO bound operation. This method is for special cases where adding needs to access the database

Reply:
TIL